.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       matrix:
         version:
           - '1.0'
-          - '1.5'
+          - '1.6'
```

Comment:
We should set this to '1', that will automatically use the latest Julia release.